### PR TITLE
[docs] Fix the code blocks in pipes.py

### PR DESCRIPTION
**Pull request by Hexcles**
Some code blocks were previously not correctly indented or surrounded
with empty lines.

### CODE REVIEW

**Comment by jugglinmike** [comment]
Unlike the well-formed example, I'm seeing the newly-indented code examples rendered as block quotes, not code:

2019-07-09-docs-pipe-formatting

Do you see code blocks on your build, @Hexcles?

**Comment by Hexcles** [comment]
Hmm, they do indeed show as blockquotes. I wonder what I got wrong? Is it because these blocks are single-line?

**Comment by Hexcles** [comment]
Oh wait a second, that double colon before the block seems to be the key... Weird syntax :(

File: tools/wptserve/wptserve/pipes.py [hunk 1]
```diff
@@ -369,16 +371,15 @@ def sub(request, response, escape_type="html"):
       {{domains[www]}} => www.localhost
       {{ports[http][1]}} => 81
 
+    It is also possible to assign a value to a variable name, which must start
+    with the $ character, using the ":" syntax e.g.
```
Comment (Hexcles) [inline tools/wptserve/wptserve/pipes.py]:
```suggestion
    with the $ character, using the ":" syntax e.g.::
```

File: tools/wptserve/wptserve/pipes.py [hunk 2]
```diff
@@ -359,6 +360,7 @@ def sub(request, response, escape_type="html"):
       sha224, sha256, sha384, and sha512. For example:
```
Comment (Hexcles) [inline tools/wptserve/wptserve/pipes.py]:
```suggestion
      sha224, sha256, sha384, and sha512. For example::
```

**Comment by jugglinmike** [comment]
Good catch. And ironic, given the surrounding content

File: tools/wptserve/wptserve/pipes.py [hunk 3]
```diff
@@ -350,6 +350,7 @@ def sub(request, response, escape_type="html"):
       For example:
```
Comment (Hexcles) [inline tools/wptserve/wptserve/pipes.py]:
```suggestion
      For example::
```

File: tools/wptserve/wptserve/pipes.py [hunk 4]
```diff
@@ -369,16 +371,15 @@ def sub(request, response, escape_type="html"):
...
+    with the $ character, using the ":" syntax e.g.
 
-    It is also possible to assign a value to a variable name, which must start with
-    the $ character, using the ":" syntax e.g.
-
-    {{$id:uuid()}}
+      {{$id:uuid()}}
 
     Later substitutions in the same file may then refer to the variable
     by name e.g.
```
Comment (Hexcles) [inline tools/wptserve/wptserve/pipes.py]:
```suggestion
    by name e.g.::
```

**Review by jugglinmike — APPROVED** [review]
Great!

**Pull request merged**